crates/release_plz_core/src/next_ver.rs

```diff
@@ -578,6 +594,18 @@ fn get_changelog(
     Ok(changelog)
 }
 
+fn get_package_path(
+    package: &Package,
+    repository: &Repo,
+    project_root: &Path,
+) -> PathBuf {
+    let package_path = package.package_path().unwrap();
```

Comment:
```suggestion
    let package_path = package.package_path().context("error while retrieving package_path: project root not found")?;
```
Can you run `cargo fmt` after accepting these changes?

Reply:
done

Reply:
I still see the unwrap here D: